deny.toml

```diff
@@ -92,20 +96,19 @@ ignore = [
 # See https://spdx.org/licenses/ for list of possible licenses
 # [possible values: any SPDX 3.11 short identifier (+ optional exception)].
 allow = [
-  "MIT",
-  "Apache-2.0",
-  "Apache-2.0 WITH LLVM-exception",
-  "BSD-2-Clause",
-  "BSD-3-Clause",
-  "CC0-1.0",
-  "MPL-2.0",
-  "ISC",
-  "GPL-3.0",
-  "GPL-3.0 WITH Classpath-exception-2.0",
-  "OpenSSL",
-  "Unicode-DFS-2016",
-  "Zlib",
-  "Unicode-3.0",
+	"MIT",
+	"Apache-2.0",
+	"Apache-2.0 WITH LLVM-exception",
+	"BSD-2-Clause",
+	"BSD-3-Clause",
+	"CC0-1.0",
+	"MPL-2.0",
+	"ISC",
+	"GPL-3.0",
+	"GPL-3.0 WITH Classpath-exception-2.0",
+	"OpenSSL",
+	"Unicode-DFS-2016",
+	"Zlib",
```

Comment:
confirming the intent to remove the `Unicode-3.0` license?

Reply:
Yup. Wasn't needed